src/goto-instrument/cover.h

```diff
@@ -22,9 +22,17 @@ class message_handlert;
 class cmdlinet;
 class optionst;
 
-#define OPT_COVER "(cover):"
-#define HELP_COVER \
-  " --cover CC                   create test-suite with coverage criterion CC\n"
+#define OPT_COVER                                                              \
+  "(cover):"                                                                   \
+  "(cover-failed-assertions)"
```

Comment:
:pick: If we keep this as a separate option from `--cover` then we should check that `--cover-failed-assertions` is not passed without `--cover` and exit with a suitable error message in the case of that particular invalid usage.